src/components/Cluster/ClusterDetail/UpgradeClusterModal.js

```diff
@@ -212,15 +218,16 @@ class UpgradeClusterModal extends React.Component {
     this.setState(
       {
         loading: true,
-        page: 'closing',
       },
       () => {
         const targetReleaseVersion = this.props.targetRelease.version;
 
         this.props.clusterActions
-          .clusterPatch(this.props.cluster, {
-            release_version: targetReleaseVersion,
-          })
+          .clusterPatch(
+            this.props.cluster,
+            { release_version: targetReleaseVersion },
+            true
```

Comment:
This makes the cluster details update after triggering the upgrade => makes our `Upgrade Available` label switch to the `Upgrade in progress...` label